Sources/Crypto/BCrypt/BCryptDigest.swift

```diff
@@ -23,15 +134,47 @@ public final class BCryptDigest {
     ///             The salt must be 16-bytes.
     /// - throws: `CryptoError` if hashing fails or if data conversion fails.
     /// - returns: BCrypt hash for the supplied plaintext data.
-    public func hash(_ plaintext: LosslessDataConvertible, cost: Int = 12, salt: LosslessDataConvertible? = nil) throws -> Data {
-        let config = try BCryptConfig(
-            version: .two(.y),
-            cost: cost,
-            salt: salt?.convertToData() ?? CryptoRandom().generateData(count: 16)
+    public func hash(_ plaintext: String, cost: UInt = 12, salt: String) throws -> String {
```

Comment:
hash methods should continue to accept `LosslessDataConvertible` (to which `String` conforms)